bundles/org.openhab.binding.fronius/README.md

```diff
@@ -20,28 +20,30 @@ The binding has no configuration options, all configuration is done at Thing lev
 
 The thing has a few configuration parameters:
 
-| Parameter | Description                                                              |
-|-----------|------------------------------------------------------------------------- |
-| Ip        | the ip-address of your Fronius Datalogger |
-| DeviceId  | The identifier of your device ( Default: 1) |
-| refresh   | Refresh interval in seconds |
+| Parameter       | Description                                           |
+|-----------------|------------------------------------------------------ |
+| hostname        | The ip-address of your Fronius Datalogger |
```

Comment:
is it possible to add a hostname here? Or are only IP addresses supported?

Reply:
As far as I can see it in the code it should be possible to use a hostname too. It just replaces the configuration value into the URL. Also the reachability check should work with a hostname.
Internally the variables are sometimes called `ip` and sometimes `hostname`...
(In my opinion it would habe been better to simply name it `host`.)

Reply:
That should be refactored, but is out of scope of this PR. Can you adjust the two open pointa in the documentation?

Reply:
Did some more cleanups of the documentation, hope it is ok this way.